systemProperties.adoc

```diff
@@ -0,0 +1,22 @@
+== System Properties
+Below, a number of relevant System Properties are listed.
+
+.System properties
+[cols=4*, options="header"]
+|===
+| Property Key | Default | Description | Java Variable Name (Type)
+| chronicle.network.callOnMissedHeartbeatOnDisconnect |`false` | If 'true', calls HeartBeatListener when we do not receive a heartbeat | _CALL_MISSED_HEARTBEAT_ON_DISCONNECT_ (boolean)
```

Comment:
If `true` or set to "", ....

Reply:
I don't think we should, let's just say if set